### PR TITLE
chore: Update CSV description for Serving, Eventing and the client.

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

**Comment by rhuss** [comment]
@mgencur @markito @markusthoemmes If this sounds correct and we can merge I will create a backport PR for release-1.11

File: templates/csv.yaml [hunk 1]
```diff
@@ -125,47 +125,36 @@ spec:
...
-       can express an interest in an event or class of events that is not yet being
-       produced.
+    Knative Eventing supports the following architectural cloud-native concepts:
+
+    - Services are loosely coupled during development and deployed independently to production
+    - A producer can generate events before a consumer is listening, and a consumer can express an interest in an event or class of events that are not yet being produced.
     - Services can be connected to create new applications
        * without modifying producer or consumer, and
        * with the ability to select a specific subset of events from a particular
-         producer.
```
Comment (lance) [inline templates/csv.yaml:152]: Did you intend to delete this word? The preceding bullet doesn't make sense without it.

**Review by lance — COMMENTED** [review]
LGTM - with one question

File: templates/csv.yaml (hunk 1 above)
Comment (rhuss) [reply templates/csv.yaml:152]: good catch. Indeed a c&p error.

**Review by lance — APPROVED** [review]

**Comment by markusthoemmes** [comment]
This needs a regeneration.

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -125,47 +125,36 @@ spec:
...
-    - Growing support Knative eventing, closely following its development
-      (managing of sources & triggers)
-    - A plugin architecture similar to that of kubectl plugins
-    - A thin client-specific API in golang which helps in tasks like synchronously
-      waiting on Knative service write operations.
-    - An easy integration of Knative into Tekton Pipelines by using kn in a Tekton
-      Task.
+    - Full support for managing all features of Knative Serving: Services, Revisions and Routes
+    - Support for managing Knative Eventing entities: Sources, Brokers, Triggers, Channels and Subscriptions
+    - A kubetl-like plugin architecture to extend the built-in functionality
```
Comment (mgencur) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:155]: `kubetl-like` -> `kubectl-like` ?

**Review by zroubalik — COMMENTED** [review]
~~Is it just me, or is the functions part quite minimal. What I mean, that at first sight I barely noticed that some functions related stuff was added.~~

Please disregard, sorry for the confusion.

**Comment by mgencur** [comment]
Suppose it would be done through a separate PR? It would make sense to have this single sentence in the client section and then a whole paragraph about functions. Like the one in https://docs.google.com/document/d/1VACc5e-G5tDh5NiZXlxV2Ekds2E42MxHFxCoSUDAewI/edit

**Comment by zroubalik** [comment]
@mgencur yeah, sorry. There was a misunderstanding between me and Roland :)

**Comment by zroubalik** [comment]
@nainaz @lance I discussed this with Roland and we decided to put the Functions part in this PR as well, PTAL

**Review by zroubalik — COMMENTED** [review]
LGTM, just the minor typo

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (rhuss) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml]: fixed

**Review by zroubalik — APPROVED** [review]
Looking good

**Pull request merged**